tests/test_audiofile.py

```diff
@@ -13,24 +13,51 @@
 import audiofile as af
 
 
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+ASSETS_DIR = os.path.join(SCRIPT_DIR, 'assets')
+
+
 @pytest.fixture(scope='function')
-def empty_wav_file(request):
-    """fixture to generate to """
+def empty_file(tmpdir, request):
+    """Fixture to generate empty audio files.
 
-    tmp_path = tempfile.mktemp('.wav', prefix="empty-audio-")
-    af.write(tmp_path, np.array([[]]), 16000)
+    The request parameter allows to select the file extension.
 
+    """
+    # Create empty audio file
+    empty_file = os.path.join(tmpdir, 'empty-file.wav')
+    af.write(empty_file, np.array([[]]), 16000)
+
+    # Rename to match extension
     file_ext = request.param
-    ofpath = audeer.replace_file_extension(tmp_path, file_ext)
-    if os.path.exists(tmp_path):
-        os.rename(tmp_path, ofpath)
+    ofpath = audeer.replace_file_extension(empty_file, file_ext)
+    if os.path.exists(empty_file):
+        os.rename(empty_file, ofpath)
 
     yield ofpath
 
     if os.path.exists(ofpath):
         os.remove(ofpath)
 
 
+@pytest.fixture(scope='function')
+def broken_file(tmpdir, request):
+    """Fixture to generate broken audio files.
+
+    The request parameter allows to select the file extension.
+
+    """
+    # Create broken audio file
+    file_ext = request.param
+    broken_file = os.path.join(tmpdir, f'broken-file.{file_ext}')
+    open(broken_file, 'w').close()
```

Comment:
Maybe a one-line comment explaining why opening and closing straight after opening results in a broken file. Good to know though  ...

Reply:
It does not result in a broken file, but a non audio file. So maybe I should change the name of the fixture as we don't create broken files (I don't know how to do this).

Reply:
> we don't create broken files (I don't know how to do this).

you could create a valid wav file and then manually change some bytes, but I would say an empty file is just as good :)

Reply:
I renamed the fixture to `non_audio_file` and updated the comment to

```python
# Create empty file to simulate broken/non-audio file
```